scripts/notify.js

```diff
@@ -39,7 +39,7 @@ async function main() {
       .filter((entry) => !beforeCacheVersions.includes(entry.version))
       .map((entry) => {
         return {
-          version: entry.version,
+          version: entry.version.replace(/^v/, ""),
```

Comment:
it's much easier to add a "v" than to remove it. Plus the other versions don't have the "v" prefix either. 

Nobody but me has the app installed yet, we are safe to make the change